src/perl/lib/wtsi_clarity/mq/messages/flowcell/sample.pm

```diff
@@ -0,0 +1,110 @@
+package wtsi_clarity::mq::messages::flowcell::sample;
+
+use Moose;
+use MooseX::Storage;
+
+our $VERSION = '0.0';
+
+with Storage( 'traits' => ['OnlyWhenBuilt'],
+              'format' => 'JSON',
+              'io' => 'File' );
+
+my @defaults = ( is => 'ro', isa => 'Str', required => 1 );
+
+has [
+     'tag_sequence',
+     'tag_set_name',
+     'pipeline_id_lims',
+     'entity_type',
+     'bait_name',
+     'sample_uuid',
+     'cost_code',
+     'entity_id_lims',
+     'is_r_and_d',
+    ] => @defaults;
+
+has 'study_uuid' => @defaults, required => 0;
+
+has [
+     'tag_index',
+     'requested_insert_size_from',
+     'requested_insert_size_to'
+    ] => @defaults, isa => 'Int';
+
+1;
+
+__END__
+
+=head1 NAME
+
+wtsi_clarity::mq::messages::flowcell::sample
+
+=head1 SYNOPSIS
+
+  my $sample = wtsi_clarity::mq::messages::flowcell::sample->new(
+    tag_sequence               => 'ATAG',
+    tag_set_name               => 'Sanger_168tags - 10 mer tags',
+    pipeline_id_lims           => 'GCLP',
+    entity_type                => 'library_indexed',
+    bait_name                  => 'DDD_V5_plus',
+    sample_uuid                => '00000000-0000-0000-000000000',
+    study_uuid                 => '00000000-0000-0000-000000001',
```

Comment:
No `study_uuid` definition in the module. If it is unneeded, then please remove it from the other synopsis, too.